prow/cluster/BUILD.bazel

```diff
@@ -32,6 +32,7 @@ release(
     component("prowjob", "customresourcedefinition"),
     component("pushgateway", "deployment"),
     component("sinker", "deployment"),
+    component("statusreconciler", "deployment"),
```

Comment:
I see `build` and `crier` use RBAC -- I don't fully understand when we can and cannot -- should I use it  for `status-reconciler` too?

Reply:
I would assume we can everywhere so long as it's properly configured?

Reply:
@cjwagner had some comments about turning off legacy authorization on the cluster?

Reply:
I think our cluster is ignoring RBAC because legacy authorization is enabled and I think it needs to be enabled until we figure out how to configure RBAC correctly for `plank`, `deck`, and `sinker` to be authenticated and authorized to create/list/delete pods in separate build clusters.  I'm not entirely certain about either of those points though.